Explorer/Assets/Plugins/DCLCrypto/Identity.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Nethereum.Signer;
+
+namespace DCLCrypto
+{
+    public class Identity
+    {
+        private readonly EthECKey key;
+        private readonly EthereumMessageSigner signer = new ();
```

Comment:
If `EthereumMessageSigner` is stateless you should have a single instance instead of creating a signer for each identity. Moreover, it looks like `singer` should not belong `Identity`